code/game/turfs/simulated/floor/transparent.dm

```diff
@@ -85,6 +87,36 @@
 	playsound(src, 'sound/items/deconstruct.ogg', 80, TRUE)
 	ChangeTurf(/turf/simulated/floor/plating)
 
+/turf/simulated/floor/transparent/glass/extinguish_light(force)
+	light_power = 0
+	light_range = 0
+	update_light()
+	name = "dimmed glass flooring"
+	desc = "Something is covering the glass, you look at it, it looks back. Perhaps a light will dim it's life?"
```

Comment:
```suggestion
	desc = "Something shadowy moves to cover the glass. Perhaps shining a light will force it to clear?"
```

This is a little confusing as a description, perhaps it can be reworded?

Reply:
maybe a little too poet-y, yeah